### DISCUSSION TITLE
Using phalcon 3.7 with php 8.2

### DISCUSSION THREAD

**Discussion by amarpolAV** [phalcon/cphalcon#16330] [General]
We have product designed in phalcon 3.7 with php 7.2 and we are upgrading to php 8.2. Is phalcon 3.7 supports php8.2 ?

**Comment by niden** [comment]
No. v5 offers support for PHP 8.0+

v3 has not been supported for years :/

**Comment by burgeeblue** [comment]
No, v5 supports PHP 8.0 and higher. 
https://github.com/orgs/phalcon/discussions/16330 [geometry dash](https://geometrydash.io/)
